.circleci/config.yml

```diff
@@ -984,6 +984,33 @@ jobs:
             # Upload to GitHub
             ./ghr -replace ${CIRCLE_TAG} glean-core/python/dist
 
+  ###########################################################################
+  # C#
+
+  CSharp tests:
+    docker:
+      - image: mcr.microsoft.com/dotnet/core/sdk:3.1-buster
+    steps:
+      - checkout
+      - skip-if-doc-only
+      - install-rustup
+      - setup-rust-toolchain:
+          rust-version: "1.43.0"
+      - run:
+          name: Install the dependencies
+          command: |
+            apt update
+            apt install -y gcc
+      - run:
+          name: Build glean_ffi
+          command: cargo build
```

Comment:
It's not automatically build as part of the dotnet buildstep? :(

Reply:
No, not yet :( We'll cross that bridge after we get out of the "MOVEEEEEE FASTTTTTTTTTT" 🐎 🐎  phase :)